package.json

```
@@ -20,7 +20,8 @@
     "start": "next start",
     "postinstall": "is-ci || (husky install .husky)",
     "check-all": "npm-run-all prettier lint:fix tsc",
-    "clean": "rimraf node_modules && yarn"
+    "clean": "rimraf node_modules && yarn",
+    "update-template-tables": "python scripts/md-tables-to-json.py"
```

```suggestion
    "update-template-tables": "python3 scripts/md-tables-to-json.py"
```
See: https://stackoverflow.com/questions/64801225/python-or-python3-what-is-the-difference